### PR TITLE
Pages

### PR DESCRIPTION
**Pull request by Chakravarthi0**
added the following static pages:
product listing page
cart page
wishlist page
signin page
signup page

### CODE REVIEW

File: src/components/price-details/PriceDetails.jsx [hunk 1]
```diff
@@ -0,0 +1,38 @@
+import React from "react";
+import "./price-details.css";
+
+function PriceDetails() {
+  return (
+    <div className="price-details">
+      <h2>Price details</h2>
+      <hr />
+      <br />
```
Comment (gopalbbharadva) [inline src/components/price-details/PriceDetails.jsx:9]: Suggestion: Dont use "br" element, once tanya bhaiya told.

**Review by gopalbbharadva — APPROVED** [review]
Good to go.

**Review by manojasarsa — APPROVED** [review]
Good to merge!

**Review by msdeshmukh009 — APPROVED** [review]
The code is looking good.
You have broken down the components into pieces very well.
Good to merge.

**Pull request merged**